app/services/configs/course_information.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Configs
+  class CourseInformation
+    def initialize(course)
+      @db = Rails.application.config_for(:course_information)
+      @course = course
+    end
+
+    def show_placement_guidance?(type)
+      case type
+      when :provider_type
+        subtype = @course.provider.provider_type
+      when :program_type
+        subtype = @course.program_type
+      end
```

Comment:
Should we not handle an `else` branch here even if there are currently only two possible types? The next line requires a `subtype` value.

Reply:
That's validating the `type` argument then, isn't it? I might consider that in a complicated method that accepts various types like String, Array or Numeric for the same argument, but I'm not inclined to do that here.